test-lib.sh

```diff
@@ -1069,4 +1054,34 @@ ct_test_app_dockerfile() {
   return $ret
 }
 
+
+# ct_run_tests_from_testset
+# -----------------------------
+# Runs all tests in $TEST_SET, prints result to
+# the $TEST_SUMMARY variable
+# Argument: app_name - application name to log
+# Uses: $TEST_SET - set of test cases to run
+# Uses: $TEST_SUMMARY - variable for storing test results
+# Uses: $IMAGE_NAME - name of the image being tested
+ct_run_tests_from_testset() {
+  local app_name="$1"
+  local testsuite_result=0
+  for test_case in $TEST_SET; do
+    info "Running test $test_case ... "
+    $test_case
+    local test_result=$?
+    local test_msg
+    local output_format="%s %s $1\n"
```

Comment:
output_format is not used, so let's remove it.

Reply:
Sorry, I've missed that. Resolved.